language/move-ir-compiler/transactional-tests/tests/signer/signer_keyword.mvir

```diff
@@ -0,0 +1,7 @@
+//# print-bytecode
+module 0x0.M {
+    // "signer" is a keyword so it cannot be used as a parameter name
+    foo(signer: address) {
+        return;
```

Comment:
Sorry, since I landed #9218, you'll now need to update most of these IR tests. The Move IR syntax has changed such that this will require a label:

```suggestion
    foo(signer: address) {
    label b0:
        return;
```

Reply:
Good catch! I fixed a bunch of cases similar to this after rebasing onto your diff, but apparently missed this one. Let me do a quick search and see if there are other ones to fix.